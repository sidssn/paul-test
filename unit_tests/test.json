[{"project_id": "9f564a81-e40c-11e9-bc4f-acb57d6c5605", "project_group": "Vouliagmeni", "environments": [{"environment": "Integration"}, {"environment": "QA"}, {"environment": "UAT"}, {"environment": "Live"}], "releases": [{"version": "0.0.1", "deployments": [{"environment": "Integration", "created": "2016-03-02T14:25:00.000Z", "state": "Failed", "name": "Deploy to Integration"}]}, {"version": "0.0.2", "deployments": [{"environment": "Integration", "created": "2015-02-22T22:41:00.000Z", "state": "Success", "name": "Deploy to Integration"}]}, {"version": "0.0.3", "deployments": [{"environment": "Integration", "created": "2015-02-23T11:01:04.000Z", "state": "Success", "name": "Deploy to Integration"}]}, {"version": "0.0.4", "deployments": [{"environment": "Integration", "created": "2015-02-24T13:04:04.000Z", "state": "Success", "name": "Deploy to Integration"}]}, {"version": "0.0.5", "deployments": [{"environment": "Integration", "created": "2015-02-24T16:34:03.000Z", "state": "Success", "name": "Deploy to Integration"}, {"environment": "QA", "created": "2015-02-24T16:40:04.000Z", "state": "Failed", "name": "Deploy to QA"}]}, {"version": "0.0.6", "deployments": [{"environment": "Integration", "created": "2015-02-24T16:34:03.000Z", "state": "Success", "name": "Deploy to Integration"}, {"environment": "QA", "created": "2015-02-24T16:40:04.000Z", "state": "Failed", "name": "Deploy to QA"}]}, {"version": "0.0.7", "deployments": [{"environment": "Integration", "created": "2015-02-24T16:56:01.000Z", "state": "Success", "name": "Deploy to Integration"}, {"environment": "QA", "created": "2015-02-24T16:59:01.000Z", "state": "Success", "name": "Deploy to QA"}]}, {"version": "1.0.22.32", "deployments": [{"environment": "Integration", "created": "2015-02-25T08:57:04.000Z", "state": "Success", "name": "Deploy to Integration"}, {"environment": "QA", "created": "2015-02-25T09:00:01.000Z", "state": "Success", "name": "Deploy to QA"}, {"environment": "UAT", "created": "2015-02-25T14:28:02.000Z", "state": "Failed", "name": "Deploy to UAT"}, {"environment": "UAT", "created": "2015-02-25T14:31:01.000Z", "state": "Failed", "name": "Deploy to UAT (#2)"}, {"environment": "UAT", "created": "2015-02-25T14:33:01.000Z", "state": "Failed", "name": "Deploy to UAT (#3)"}, {"environment": "UAT", "created": "2015-02-25T14:34:03.000Z", "state": "Failed", "name": "Deploy to UAT (#4)"}, {"environment": "UAT", "created": "2015-02-25T14:45:05.000Z", "state": "Failed", "name": "Deploy to UAT (#5)"}, {"environment": "UAT", "created": "2015-02-25T14:58:02.000Z", "state": "Success", "name": "Deploy to UAT (#6)"}, {"environment": "Live", "created": "2015-02-26T13:59:02.000Z", "state": "Success", "name": "Deploy to Live"}]}, {"version": "1.0.24.34", "deployments": [{"environment": "Integration", "created": "2015-02-28T21:55:01.000Z", "state": "Success", "name": "Deploy to Integration"}, {"environment": "QA", "created": "2015-02-28T22:02:02.000Z", "state": "Success", "name": "Deploy to QA"}, {"environment": "UAT", "created": "2015-03-03T14:30:00.000Z", "state": "Failed", "name": "Deploy to UAT"}, {"environment": "Integration", "created": "2015-03-03T14:45:02.000Z", "state": "Success", "name": "Deploy to Integration (#2)"}, {"environment": "Integration", "created": "2015-03-03T14:50:05.000Z", "state": "Success", "name": "Deploy to Integration (#3)"}, {"environment": "UAT", "created": "2015-03-03T14:59:04.000Z", "state": "Success", "name": "Deploy to UAT (#2)"}, {"environment": "Live", "created": "2015-03-04T11:03:04.000Z", "state": "Success", "name": "Deploy to Live"}]}]}]