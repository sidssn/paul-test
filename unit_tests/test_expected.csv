Day,TestCount
Monday,2
Tuesday,3
